src/App/pages/ProductDetail/ProductDetail.tsx

```diff
@@ -85,6 +74,6 @@ const ProductDetail = () => {
       )}
     </div>
   );
-};
+});
 
 export default ProductDetail;
```

Comment:
```suggestion
export default observer(ProductDetail);
```
Можно обернуть здесь. Мне кажется, так красивее